src/AnalyticsClientFactory.php

```diff
@@ -21,18 +21,11 @@ public static function createAuthenticatedGoogleClient(array $config): Google_Cl
     {
         $client = new Google_Client();
 
-        $client->setClassConfig(
-            'Google_Cache_File',
-            'directory',
-            $config['cache_location'] ?? storage_path('app/laravel-google-analytics/google-cache/')
-        );
-
-        $credentials = $client->loadServiceAccountJson(
-            $config['service_account_credentials_json'],
-            'https://www.googleapis.com/auth/analytics.readonly'
-        );
-
-        $client->setAssertionCredentials($credentials);
+        $client->setScopes([
```

Comment:
I'm still figuring out, `setClassConfig ` and even `Google_Cache_File` are no longer available.